app/app/settings.py

```diff
@@ -31,6 +31,15 @@
 RATELIMIT_USE_CACHE = 'default'
 RATELIMIT_VIEW = 'tdi.views.ratelimited'
 
+# Gitcoinbot Account Information
+GITHUB_API_USER = os.environ.get('GITHUB_API_USER')
+GITHUB_API_TOKEN = os.environ.get('GITHUB_API_TOKEN')
+
+# Gitcoinbot APP Information
+GITCOINBOT_APP_ID =  os.environ.get('GITCOINBOT_APP_ID')
```

Comment:
We have a PR out that switches all settings over to env vars.  Can we stick with the current format for the time being?